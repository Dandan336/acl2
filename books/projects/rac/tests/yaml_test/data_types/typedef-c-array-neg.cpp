// RAC begin

typedef int fixed_size_arr[-4];

int foo()
{
  return 0;
}

// RAC end
