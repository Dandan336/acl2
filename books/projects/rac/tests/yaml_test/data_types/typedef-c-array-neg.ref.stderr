typedef-c-array-neg.cpp:3: Array dimension not a positive integer constant
